wagtail/core/blocks/list_block.py

```diff
@@ -65,7 +84,7 @@ def clean(self, value):
         if any(errors):
             # The message here is arbitrary - outputting error messages is delegated to the child blocks,
             # which only involves the 'params' list
```

Comment:
This comment (and the corresponding one on StructBlock) is made obsolete by this change, and can be deleted.